lib/commands/general.js

```diff
@@ -95,6 +95,25 @@ commands.mobileNavigateTo = async function (opts = {}) {
   });
 };
 
+/**
+ * Runs a chain of Espresso web atoms
+ *
+ * Takes JSON of the form
+ *
+ * {
+ *   "webviewEl": "<ELEMENT_ID>", // optional webview element to operate on
+ *   "forceJavascriptEnabled": true|false, // if webview disables javascript, webatoms won't work, this forces it
+ *   "methods": [
+ *     {"name": "methodName", "atom": {"name": "atomName", "args": ["arg1", "arg2", ...]}},
```

Comment:
Would you mind to also add a link containing a list of available atoms?

Reply:
Done